.github/workflows/main.yml

```
@@ -36,20 +35,8 @@ jobs:
-
-    - name: Setup .NET
-      uses: actions/setup-dotnet@v4
-      with:
-        global-json-file: ./global.json
-
-    - name: NuGet Restore
-      run: dotnet restore
-
-    - name: Build
-      run: dotnet build --no-restore --configuration Release /p:Deterministic=true /p:UseSharedCompilation=false /p:BuildInParallel=false /nodeReuse:false /bl:./artifacts/logs/release/build.release.binlog
+    - name: Setup, Restore, and Build Solution
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)